reference/user_defined_functions.rst

```diff
@@ -32,6 +32,8 @@ Arguments
 **distribution_method:** (Optional) The method according to which the table is
 to be distributed. Permissible values are append or hash, and defaults to 'hash'.
 
+**colocate_with:** (Optional) include current table in the colocation group of another table. By default tables distributed by columns of the same type and with the same shard count are co-located. (See :ref:`colocation_groups`.)
```

Comment:
Should we document the various options here? There are some special strings which results in specific behavior ('default' and 'none'). There is also the behavior when no option is specified (same as specifying 'default')